### PR TITLE
add instructions for connecting to eduroam on android

### PR DESCRIPTION
**Pull request by tnt7864**
an intro section may be needed? but i got the instructions in here at least

### CODE REVIEW

**Comment by MoralCode** [comment]
Thanks for contributing this! #9 just got merged containing a pretty major reorganisation of the eduroam pages. If you have time, could you update your pull request to be based on the current main branch? if you don't have just let me know and I can also help make these fixes.

This should simplify things a lot since there's already some examples of tutorials for other networking software that have been made into separate pages and link to a single "how to get your certificates" guide to deduplicate things a bit.

Thanks again for the contribution!

**Comment by tnt7864** [comment]
done!

File: eduroam/android.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+2. Clicking 'Install certificates' should open up a file picker. Navigate to your user certificate, which is likely in your 'Downloads' folder, and choose that. You will need to enter the password you chose earlier for the certificate, and then enter a name for the certificate. Make sure under 'Credential use' you pick 'Wi-Fi'.  
+   ![The Android file picker, showing 'Downloads' folder and a file called '{blacked out} (RIT Student).p12'](/assets/img/eduroam-android/downloads-p12.png)
+   ![A dialog titled 'Extract certificate' with a password prompt](/assets/img/eduroam-android/cert-password.png)
+   ![A dialog titled 'Name the certificate' with a textbox for the name and a dropdown called 'Crdential use', with the options 'VPN and apps' and 'Wi-Fi'](/assets/img/eduroam-android/rit-user-cert-name-dialog.png)
+3. Do the same for the RIT root CA.  
+   > Some phones (like Samsung) won't allow you to install CA certificates without root, and if so, you may skip this step. Having the RIT root CA installed is recommended but not necessary.
+
```
Comment (MoralCode) [inline eduroam/android.md:26]: how are these blockquotes intended to be formatted? they stand out a bit differently from the rest of the text. is that intentional?

**Review by MoralCode — APPROVED** [review]
Looks good! i made some minor tweaks to fix some small stuff. Also thanks for the fixes you made to correct typos and other small errors on the site!

Ideally it would be nice to have the images inside a subfolder under the eduroam images folder to match the same structure, but thats more of a minor nitpick than anything and IMO shouldn't prevent this from being merged. LMK if you do or dont intend to fix this so i know when i should hit merge.

Heres a screenshot of how the page looks as rendered (see my comment inline) in case you havent had a chance to run it through jekyll:

image

**Comment by MoralCode** [comment]
oh, one more question i had was regarding the root cert. does samsung only require root access for installing _root certificates_, or any certificates? since you mentioned this in the root certs step, it might be good to clarify in the step before it with what the expectation is regarding the normal cert

File: eduroam/android.md (hunk 1 above)
Comment (tnt7864) [reply eduroam/android.md:26]: the intention was an aside or tip, slightly outside the regular flow of text so it doesn't blend in with everything else
the formatting is more intense and out of place than intended, i agree. i'll replace it with just a "Note:" prefix where needed, i think that's more of what I was aiming for anyways

**Comment by tnt7864** [comment]
just the ca certs
it was secondhand knowledge for me, though, so i looked it up and apparently modern samsung phones *do* let you install ca certs, but [hide it in an obscure settings menu separate from user cert installation](https://stackoverflow.com/a/65319223). i'll change the wording along with the folder structure for the images

**Comment by MoralCode** [comment]
> just the ca certs
> it was secondhand knowledge for me, though, so i looked it up and apparently modern samsung phones *do* let you install ca certs, but [hide it in an obscure settings menu separate from user cert installation](https://stackoverflow.com/a/65319223). i'll change the wording along with the folder structure for the images

Seems like it might also be helpful to add that stackoverflow post detailing how to add the root CA cert

**Pull request merged**